README.md

```diff
@@ -48,31 +48,36 @@ The following environment variables need to be configured for **GIDS**:
 2. `ADMIN_EMAIL`: This is the email you will use to sign onto **GIDS**.
 3. `ADMIN_PW`: This is the password for the email (above) you will use.
 4. `LINK_HOST`: This will be `http://localhost:3000`
+5. `GOVDELIVERY_STAGING_SERVICE`: This is 'True' or 'False' and a string since they are set by python.
+6. `GOVDELIVERY_TOKEN`: This is the token for govdelivery.com.
+7. `GOVDELIVERY_URL`: This is the URL with which we send devise emails.
```

Comment:
In GH, you can just use `1.` for every line, and it'll auto-increment as an ordered list

Reply:
I was planning on doing that actually! Unfortunately, I didn't think that would work since we have paragraphs in-between